index.js

```diff
@@ -142,26 +142,26 @@ function formatSettings(settings) {
 	if (options.list && options.markdown)
 		s += "```\n";
 
-	// append explaination
+	// append explanation
 	if (options.explain && !options.list) {
 		if (options.markdown) {
 			s += "```css\n";
 			s += "/* this is how this channel is organized */\n" +
-			     "Category {\n" +
-			     "    module.m.css {\n" +
-			     "        /* list of options and modifications in the form of variables */\n" +
-			     "    }\n" +
-			     "}\n\n" +
-			     "/* your import link would then be `https://customa.gitlab.io/Customa-Discord/Category/module.m.css\` */";
+				"Category {\n" +
+				"    module.m.css {\n" +
+				"        /* list of options and modifications in the form of variables */\n" +
+				"    }\n" +
+				"}\n\n" +
+				"/* your import link would then be `https://customa.gitlab.io/Customa-Discord/Category/module.m.css\` */";
```

Comment:
align using spaces not tabs pls

Reply:
you: use tabs
also you: ^^

Reply:
also that would mean I have to format the entire thing with spaces; it be fucky like that